core/src/data/value/error.rs

```diff
@@ -23,6 +23,9 @@ pub enum ValueError {
     #[error("failed to parse number")]
     FailedToParseNumber,
 
+    #[error("failed to convert Float to Decimal: {0}")]
+    F64ToDecimalConversionError(f64),
```

Comment:
<https://github.com/gluesql/gluesql/blob/c49073a76e7a775a8a64752f72d8315c876de68f/core/src/data/value/error.rs#L63-L64>

1. Use `Float` instead of `F64`.
Float type is exposed to the user, but F64 is an internal type, so it is recommended to use `Float` for this error value exposed to the user.
2. Let's clear the `Error` suffix.
Declaring a variant in the ValueError enum is duplicated.
How about something like `FloatToDecimalConversionFailure` instead?

Reply:
Good suggestions!